tests/integration-tests/tests/createami/test_createami.py

```diff
@@ -57,4 +59,23 @@ def test_createami(region, os, instance, request, pcluster_config_reader, vpc_st
         + networking_args
     )
 
+    assert_that(
+        any(
+            "downloading https://{0}-aws-parallelcluster.s3".format(region).lower() in s.lower()
+            for s in pcluster_createami_result.stdout.split("\n")
+        )
+    ).is_true()
+    assert_that(
+        any("chef.io/chef/install.sh".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+    ).is_false()
+    assert_that(
+        any("packages.chef.io".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
+    ).is_false()
+    assert_that(
+        any("Thank you for installing Chef".lower() in s.lower() for s in pcluster_createami_result.stdout.split("\n"))
```

Comment:
Instead of re-executing the "for" in each assertion you can create it one time and then use for all the assertions.